alf/environments/process_environment.py

```diff
@@ -64,6 +64,64 @@ class _MessageType(Enum):
     CLOSE = 6
 
 
+def _worker(conn, env_constructor, env_id=None, flatten=False):
```

Comment:
Note that this is a full copy of the original method, **without any** modification.